windows/CodePushUtils.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.System.Profile;
+
+namespace CodePush.ReactNative
+{
+    class CodePushUtils
+    {
+        public static readonly string REACT_NATIVE_LOG_CATEGORY = "ReactNative";
+
+        public async static Task<JObject> GetJObjectFromFile(StorageFile file)
+        {
+            string jsonString = await FileIO.ReadTextAsync(file);
+            if (jsonString.Length == 0)
+            {
+                return new JObject();
+            }
+
+            return JObject.Parse(jsonString);
+        }
+
+        public static void log(string message)
+        {
+            Debug.WriteLine("[CodePush] " + message, REACT_NATIVE_LOG_CATEGORY);
+        }
+        
+        public static void logBundleUrl(string path)
+        {
+            log("Loading JS bundle from \"" + path + "\"");
+        }
+
+        public static string GetDeviceId()
+        {
```

Comment:
This is a general purpose comment: does it make sense to annotate a method as public when it's contained in an internal class? I feel like we should potentially remove the public modifiers from all internal classes, since I'm not sure it makes a ton of sense.